packages/hawtio/src/core/logging.ts

```diff
@@ -79,13 +79,15 @@ class LocalStorageHawtioLogger implements HawtioLogger {
       const logLevel = this.loadLogLevel()
       jsLogger.setLevel(logLevel)
     } catch (e) {
+      // eslint-disable-next-line no-console
```

Comment:
likewise, this is where the logging is defined so it rather makes sense to disable it for the entire file.